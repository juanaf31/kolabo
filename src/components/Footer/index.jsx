import { Col, Row, Typography } from "antd";
import React from "react";

const Footer = () => {
    return (
        <footer
            style={{
                backgroundColor: "#465A6E",
                color: "white",
                padding: "20px 50px",
            }}>
            <Row data-testid='footer' justify="end">
                <Col>
                    <Typography style={{ color: "#FBFBFBB3" }}>
                        Kolabo
                    </Typography>
                    <Typography style={{ color: "#FBFBFBB3" }}>
                        Copyright 2021 Â©
                    </Typography>
                </Col>
            </Row>
        </footer>
    );
};

export default Footer;
