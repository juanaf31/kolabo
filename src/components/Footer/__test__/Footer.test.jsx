import { cleanup, render } from '@testing-library/react'
import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../index'

afterEach(() => {
    cleanup()
})

describe('testing footer component', () => {

    test('footer matches snapshot', () => {
        const tree = renderer.create(<Footer />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})