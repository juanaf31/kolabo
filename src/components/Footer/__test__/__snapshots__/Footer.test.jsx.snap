// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing footer component footer matches snapshot 1`] = `
<footer
  style={
    Object {
      "backgroundColor": "#465A6E",
      "color": "white",
      "padding": "20px 50px",
    }
  }
>
  <div
    className="ant-row ant-row-end"
    data-testid="footer"
    style={Object {}}
  >
    <div
      className="ant-col"
      style={Object {}}
    >
      <article
        className="ant-typography"
        style={
          Object {
            "color": "#FBFBFBB3",
          }
        }
      >
        Kolabo
      </article>
      <article
        className="ant-typography"
        style={
          Object {
            "color": "#FBFBFBB3",
          }
        }
      >
        Copyright 2021 Â©
      </article>
    </div>
  </div>
</footer>
`;
